### ISSUE TITLE
Another model added to Multiple Devices

### ISSUE THREAD

**Issue by arucard21** [Quoturnix/ProjectTox-Core#25]
After looking at the models described in https://github.com/Quoturnix/ProjectTox-Core/wiki/Multiple-devices, I added another model. It uses some of the ideas of the other models, but shouldn't break existing functionality and therefore be easier to implement. I'm not very familiar with DHT so I hope that it's actually possible to do this. 

I didn't want to just edit it and not say anything. I'm not sure if it's even still relevant or if you've already implemented something for this functionality. I hope it's alright that I edited the document directly though. I just wanted to let you know I added it and I hope it's helpful.

**Comment by alnf** [comment]
We are not working on Multiple Devices right now, cause are busy with Group Chats, which main repo now is https://github.com/JFreegman/toxcore/ and https://github.com/alnf/toxcore/. And I think we are not the ones who will be implementing this, because https://wiki.tox.im/GSoC/2015/Ideas. So I'll post the link to gsoc students channel, so your effort won't be in vain:)

This repo will be eventually deleted.

**Comment by alnf** [comment]
And thank you for informing about changing the wiki, I wouldn't even know.

**Comment by arucard21** [comment]
Thank you for the information, and for relaying my idea to the gsoc students. I hope it helps.

**Comment by arucard21** [comment]
I was curious about how the Group Chats was being implemented, but I could only find this:
https://github.com/JFreegman/toxcore/blob/new_groupchats/docs/Group-Chats.md
It's four months old and rather short, is this the best available information on the design of Group Chats? I assume it's already being implemented, but I was still curious about its design. It's really interesting to see how Tox is developing its protocol from scratch.

**Comment by alnf** [comment]
No, it's not. https://github.com/alnf/toxcore/wiki/New-Group-Chats - the most fresh written information, however, it is not always up to date with the code, even though we try to keep it fresh. We closed wiki access because of that reasons partically. It is confusing when someone changes smth without notifying, considering only developers now the most recent info. Also we sometimes keep not implemented ideas in the wiki or concurrent ideas. But you are welcome to create the issues:) And you can actually test new groupchats already. Compile JFreegman branch and Jfreegman toxic new_groupchats branch.

**Comment by alnf** [comment]
You may join #tox-groupchats freenode channels for more information.

**Comment by landswellsong** [comment]
@alnf  should I axe this repo for good maybe?
